lib/framework/data_context.dart

```diff
@@ -543,6 +546,96 @@ class EnsembleStorage with Invokable {
   }
 }
 
+/// singleton handling utils
+class EnsembleUtils with Invokable {
+  static final EnsembleUtils _obj = EnsembleUtils._();
+  EnsembleUtils._();
+  factory EnsembleUtils() => _obj;
+
+  @override
+  Map<String, Function> getters() => {'getCountries': () => allCountries};
```

Comment:
You want to move this to method so you can use `utils.getCountries()`. Getters syntax will be `utils.getCountries` which is not what we want.